SPECS-EXTENDED/linuxptp/linuxptp.spec

```diff
@@ -93,15 +163,52 @@ PATH=..:$PATH ./run
 %{_sbindir}/phc_ctl
 %{_sbindir}/pmc
 %{_sbindir}/ptp4l
-%{_sbindir}/snmp4lptp
 %{_sbindir}/timemaster
+%{_sbindir}/ts2phc
 %{_mandir}/man5/*.5*
 %{_mandir}/man8/*.8*
 
 %changelog
-* Thu Oct 14 2021 Pawel Winogrodzki <pawelwi@microsoft.com> - 2.0-8
-- Initial CBL-Mariner import from Fedora 32 (license: MIT).
-- Converting the 'Release' tag to the '[number].[distribution]' format.
+* Thu Nov 16 2023 Harshit Gupta <guptaharshit@microsoft.com> - 3.1.1-1
+- Initial CBL-Mariner import from Fedora 37 (license: MIT).
+- License Verified.
+- Upstream linuxptp v3.1.1-6 has been imported into Azure Linux with package v3.1.1-1
```

Comment:
nit: should versions be consistent (either always include the 'v' prefix or never include it?